src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -729,9 +729,7 @@ public WebElement getStatusMessage() {
     }
     
     public InstructorFeedbacksPage clickDoneEditingLink() {
-        WebElement doneEditingLink = browser.driver.findElement(By.id("addNewQuestionTable"))
-                                                   .findElements(By.tagName("a"))
-                                                   .get(12);
+        WebElement doneEditingLink = browser.driver.findElement(By.id("button_done_editing"));
```

Comment:
I prefer this to be a field of `InstructorFeedbackEditPage` itself. Use the `@FindBy` annotation to make this a field.